server/error-handler.js

```diff
@@ -23,6 +29,10 @@ function handleError(error, req, res, next) {
     } else {
       logger.error(error);
     }
+  } else if (!isRippleError(error)) {
+    // always log stack traces for uncaught exceptions
+    /* eslint-disable no-unused-expressions */
```

Comment:
Many programming languages also have short-circuit evaluation. This is a stylistic preference